src/main/java/org/fundacionjala/coding/dennis/Customer.java

```diff
@@ -0,0 +1,82 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.ArrayList;
+
+/**
+ *
+ */
+public class Customer {
+    private String name;
+    private ArrayList<Rental> rentals = new ArrayList(1);
+
+    /**
+     *
+     * @param name asigned.
+     */
+    public Customer(final String name) {
+        this.name = name;
+    }
+
+    /**
+     *
+     * @param arg add rental.
+     */
+    public void addRental(final Rental arg) {
+        rentals.add(arg);
+    }
+
+    /**
+     *
+     * @return how many rentals.
+     */
+    public int numRentalElements() {
+        return rentals.size();
+    }
+
+    /**
+     *
+     * @return give me the name.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Calculus of data.
+     */
+    public void statement() {
+        float totalAmount = 0.0f;
+        float thisPrice;
+        int frequentRenterPoints = 0;
+        int thisFrequentRenterPoints;
+        System.out.println(customerName());
+        for (Rental eachRental : rentals) {
+            thisPrice = eachRental.calculatePriceMovie();
+            System.out.println(movieAndPrice(eachRental, thisPrice));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)